inputfiles/addedTypes.json

```diff
@@ -89,6 +89,120 @@
         ]
     },
     {
+        "kind": "typedef",
+        "flavor": "Worker",
```

Comment:
We will need two entries for createImageBitMap, or at least two definitions for `ImageBitmapSource`, one for WebWorkers that does not have HTMLImageElement, SVGElement, HTMLVideoElement, or HTMLCanvasElement